src/shared/lib/services/Logger.ts

```diff
@@ -38,7 +39,7 @@ type DeviceInfoLogObject = {
   lastUpdateTime: number;
 };
 
-class Logger implements ILogger {
+export class Logger implements ILogger {
```

Comment:
If this change intends to be able to mock `Logger`'s functions why can't we mock the function of the singletone in the first place?

Reply:
To test new instance of the class, not the default one. Again - tests isolation